pandas/core/indexes/base.py

```diff
@@ -151,6 +151,12 @@ class Index(IndexOpsMixin, StringAccessorMixin, PandasObject):
 
     _engine_type = libindex.ObjectEngine
 
+    _accessors = frozenset(['dt', 'str', 'cat'])
+
+    # String Methods
+    str = base.AccessorProperty(strings.StringMethods,
```

Comment:
Then should I revert deletion of `StringAccessorMixin` and mix that back in to `Index`?